src/main/kotlin/nl/info/zac/app/admin/converter/RestZaakafhandelParametersConverter.kt

```diff
@@ -127,22 +128,20 @@ class RestZaakafhandelParametersConverter @Inject constructor(
     @Suppress("ThrowsCount")
     fun toZaaktypeCmmnConfiguration(
         restZaakafhandelParameters: RestZaakafhandelParameters
-    ): ZaaktypeCmmnConfiguration {
-        val zaaktypeCmmnConfiguration = restZaakafhandelParameters.zaaktype.uuid?.let { uuid ->
-            zaaktypeCmmnConfigurationBeheerService.fetchZaaktypeCmmnConfiguration(uuid)
-        } ?: throw NullPointerException("restZaakafhandelParameters.zaaktype.uuid is null")
-
-        return zaaktypeCmmnConfiguration.apply {
+    ): ZaaktypeCmmnConfiguration =
+        zaaktypeCmmnConfigurationBeheerService.fetchZaaktypeCmmnConfiguration(
+            restZaakafhandelParameters.zaaktype.uuid
+        ).apply {
             id = restZaakafhandelParameters.id
-            zaakTypeUUID = restZaakafhandelParameters.zaaktype.uuid
+            zaaktypeUuid = restZaakafhandelParameters.zaaktype.uuid
             zaaktypeOmschrijving = restZaakafhandelParameters.zaaktype.omschrijving
                 ?: throw NullPointerException("restZaakafhandelParameters.zaaktype.omschrijving is null")
             caseDefinitionID = (
                 restZaakafhandelParameters.caseDefinition
                     ?: throw NullPointerException("restZaakafhandelParameters.caseDefinition is null")
                 ).key
-
             groepID = restZaakafhandelParameters.defaultGroepId
+                ?: throw NullPointerException("restZaakafhandelParameters.defaultGroepId is null")
```

Comment:
Brr. We are throwing our own nullpointers? Bizarre. Anyway, for a future PR I guess. ;-)